src/Features/CSharp/Portable/RemoveUnnecessaryImports/AbstractCSharpRemoveUnnecessaryImportsService.Rewriter.cs

```diff
@@ -158,7 +158,24 @@ public override SyntaxNode VisitNamespaceDeclaration(NamespaceDeclarationSyntax
 
             private static SyntaxToken StripNewLines(SyntaxToken firstToken)
```

Comment:
What about VB?

Reply:
I feel like this would be much simpler if it just did 

```c#
var trimmedLeadingTrivia = firstToken.LeadingTrivia.SkipWhile(t => t.Kind() == SyntaxKind.EndOfLineTrivia).ToList();
// now as long as the list ends with 3 newlines or more, keep removing the last newline.
```